vivo-project/VIVO
# VIVO-1451 Capability Map i18n

**This PR makes the Capability Map fully internationalized.**
* * *

**[JIRA Issue](https://jira.duraspace.org/projects/VIVO)**: https://jira.duraspace.org/browse/VIVO-1451



# What does this pull request do?
After integrating this PR, one will be able to display the feature in different languages and then run the feature in the current language selected.

# What's new?
Text from template are replaced by reference to values in language properties files.
Text in Javascrpt are replace by variables containing values from language properties files.
The SPARQL queries are language aware and retrieve only the data having the proper language tag


# How should this be tested?
This new feature should be tested using a multilingual sample dataset, here are the steps:

1- Build VIVO languages updated on this PR: https://github.com/vivo-project/VIVO-languages/pull/9
2- Build VIVO and enable fr-ca in runtime.properties
3-import sample data n3 dataset in VIVO (The corresponding PR creates transaltions for fr-CA: https://github.com/vivo-project/VIVO-languages/pull/9)
4-Select English Language and run search then Search and expand
5-Only English terms should appear among research areas and on the resulting graph
4-Select another language (fr-ca) 
6-You should be redirected on the capability map context but cleared 
7-All the interface should appear in the selected language 
8-Search then "search and expand"
9-Research areas in the search bar and result apprearing should all be in fr-CA

# Additional Notes:
This PR is comming along with PRs on VIVO-languages(https://github.com/vivo-project/VIVO-languages/pull/9) and;
VIVO-sampledata (https://github.com/vivo-project/sample-data/pull/2) and require them.


# Interested parties
@awoods @gneissone
